configure

```diff
@@ -35,21 +35,24 @@ import subprocess
 import shutil
 import string
 
+# If not run from node/, cd to that directory.
+if os.path.dirname(__file__):
+  os.chdir(os.path.dirname(__file__))
```

Comment:
You could write this as:
```python
os.chdir(os.path.dirname(__file__) or '.')
```
Which might make it more obvious that `os.path.dirname('configure') == ''`.

Or maybe that's just me.

Reply:
I like it, added.